package.json

```diff
@@ -71,5 +71,10 @@
       "pre-commit": "npm run lint && npm run test",
       "pre-push": "npm run codeclimate"
     }
+  },
+  "mocha": {
+    "timeout": 5000,
+    "checkLeaks": true,
+    "bail": true
```

Comment:
Is there an advantage moving mocha opts to within package.json? I notice you dropped the `recursive` flag. Is this now a default?

Reply:
mocha.ops is deprecated now. There are couple of other alternatives suggested but, I felt keeping it in package.json (which is one of those recommended options) makes it more simpler. I didn't find recursive flag among the set of options in their documentation. Better to recheck once and confirm.

Reply:
Thanks. Didn't know mocha.opts was deprecated. Will go with the package.json config